contributing.md

```diff
@@ -11,43 +11,55 @@ changes are not RFC-worthy.
 https://github.com/enarx/enarx/wiki/How-to-contribute),
 Encouraging feedback from maintainers is a good sign that you're on
 the right track.
 
 ### How to propose an RFC
 
-  1. Fork [the RFC repo](https://github.com/enarx/rfcs).
+  1. Fork [the RFC repo](https://github.com/enarx/rfcs). (cf. our 
+    [How to Contribute](https://github.com/enarx/enarx/wiki/How-to-contribute-RFC#git-environment-set-up)
+    document.)
   2. Choose a descriptive name for your RFC. Don't pick a number yet.
-  3. Decide which parent folder is appropriate for your RFC: `/features` or 
-    `/concepts`.
-    If it is about a specific protocol or decorator or feature, its parent
-    should be `/features`; if it is about a concept that will be used in many
-    different features, its parent should be `/concepts`.
-  4. Create a directory with the descriptive name you chose and copy
-    `00000-template.md` to `<parent>/<your descriptive RFC name>/README.md`.
-  5. Fill in the RFC (ie: fill README.md). [Use MUST and SHOULD per standard conventions](https://tools.ietf.org/html/rfc2119).
+  3. Create a directory with the descriptive name you chose (ie: `mkdir ./your_descriptive_RFC_name`)
+    Copy `00000-template.md` from the root of your forked RFC repo to `./<your_descriptive_RFC_name>/README.md`.
+  4. Fill in the RFC (ie: fill README.md). [Use MUST and SHOULD per standard conventions](https://tools.ietf.org/html/rfc2119).
     Please pay attention to details:
     RFCs that do not present convincing motivation, demonstrate an
     understanding of the impact of the design, or are disingenuous about the
     drawbacks or alternatives tend to be poorly received. You can add
     supporting artifacts, such as diagrams and sample data, in the RFC's folder.
-  6. When you are ready to publish your RFC, it is time to assign a number to
-    it. The RFC number will be the Github Pull Request number (with the the
-    necessary leading number of zeroes). To find this number, inspect open and
-    closed PRs in this repo to figure out what the next PR number will be.
-  7. Add the RFC number to your directory. Rename it from `<your descriptive RFC name>`
-    to `<your 5-digit number>-<your descriptive RFC name>`. Don't forget to
-    put the number at the top of your README.md: modify the title so it is in
-    the form `<your 5-digit number>: Friendly Version of Your Title`.  
-  8. Commit your changes.
-  9. Push your changes to your forked `rfcs` repo.
-  10. Submit a pull request from your forked repo to `enarx/rfcs`.
-
-A note on numbering: if need be, you can always create the PR to get the
-number and then modify your files to incorporate this number and push them
+  5. When you are ready to publish your RFC, it is time to assign it a number.
+    RFC number should be sequential, so to find your RFC number look at
+    latest RFCs, both [accepted](https://github.com/enarx/rfcs/blob/master/index.md)
+    and [in progress](https://github.com/enarx/rfcs/pulls)
```

Comment:
This is not accurate. To pass the `name` test, RFCs should be numbered correctly relative to only the RFCs that have been fully merged into the upstream tree.

The expectation is that this number will change if other RFCs are merged before the author's.